cdragontoolbox/wad.py

```diff
@@ -222,6 +222,17 @@ def set_unknown_paths(self, path):
                 else:
                     wadfile.path = f"{path}/{wadfile.path_hash:016x}"
 
+    def sanitize_paths(self):
+        """Truncate files that have a length of over 255"""
+
+        for wadfile in self.files:
+            path, filename = os.path.split(wadfile.path)
```

Comment:
You should check that `wadfile.path is not None`. `sanitize_path()` could have been called on a wad with unresolved paths.